src/components/MarkdownProvider/components/BestPractice.tsx

```diff
@@ -0,0 +1,177 @@
+import { MD } from '@zendeskgarden/react-typography';
+import { ReactComponent as XStrokeIcon } from '@zendeskgarden/svg-icons/src/16/x-stroke.svg';
+import { ReactComponent as CheckLgStrokeIcon } from '@zendeskgarden/svg-icons/src/16/check-lg-stroke.svg';
+import { ReactComponent as AlertErrorStrokeIcon } from '@zendeskgarden/svg-icons/src/16/alert-error-stroke.svg';
+
+import { getColor } from '@zendeskgarden/react-theming';
+
+export interface IStyledBestPracticeTextProps {
+  hue: string;
+}
+
+export interface IFileNode {
+  name: string;
+  publicURL: string;
+}
+
+export interface IFileEdge {
+  node: IFileNode;
+}
+
+const StyledBestPracticeText = styled.div<IStyledBestPracticeTextProps>`
+  border-top: ${props => `${props.theme.borders.md} ${getColor(props.hue, 500, props.theme)}`};
+  border-bottom-left-radius: ${props => props.theme.borderRadii.md};
+  border-bottom-right-radius: ${props => props.theme.borderRadii.md};
+  background: ${props => getColor('neutralHue', 100, props.theme)};
+  padding: ${props => `${props.theme.space.md}`};
+
+  ul {
+    margin: 0;
+    list-style: none;
+  }
+
+  li {
+    padding: ${props => props.theme.space.base}px 0;
+  }
+
+  li:not(:last-child) {
+    border-bottom: ${props =>
+      `${props.theme.borders.sm} ${getColor('neutralHue', 300, props.theme)}`};
+  }
+`;
+
+const StyledIcon = styled(({ children, ...props }) =>
+  cloneElement(React.Children.only(children), props)
+)`
+  margin-right: ${props => props.theme.space.xs};
+  color: ${props => getColor(props.hue, 500, props.theme)};
+`;
+
+const StyledHeaderIcon = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: ${props => props.theme.space.xs};
+`;
+
+const StyledBestPractice = styled.div`
+  display: flex;
+  justify-content: space-around;
+
+  @media (max-width: ${p => p.theme.breakpoints.md}) {
+    justify-content: space-between;
+  }
+
+  @media (max-width: ${p => p.theme.breakpoints.sm}) {
+    display: block;
+  }
+`;
+
+export const BestPractice = (props: HTMLAttributes<HTMLDivElement>) => {
+  return <StyledBestPractice {...props} />;
+};
+
+const StyledBestPracticeContainer = styled.div`
+  display: flex;
+  flex-basis: 0;
+  flex-direction: column;
+  flex-grow: 1;
+  margin-bottom: ${props => props.theme.space.xxl};
+
+  @media (min-width: ${p => p.theme.breakpoints.sm}) {
+    margin: ${props => props.theme.space.md} ${props => props.theme.space.xs};
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+`;
+
+const useAvatarFiles = () =>
+  useStaticQuery(graphql`
+    {
+      allFile(filter: { name: { regex: "/components-avatar/" } }) {
+        edges {
+          node {
+            name
+            publicURL
+          }
+        }
+      }
+    }
+  `);
+
+const staticQueries: Record<string, Function> = {
+  avatar: useAvatarFiles
+};
+
+export const BestPracticeSection: React.FC<{
+  type: 'do' | 'dont' | 'caution';
```

Comment:
Don't want to lose @austin94 [suggestion](https://github.com/zendeskgarden/website/pull/34#discussion_r423270284) of adding `<Do>`, `<Dont>` and `<Caution>` components. That makes for a nicer API.